apiplugin/file/filehost.go

```diff
@@ -75,7 +76,7 @@ func (fs FileStatus) String() string {
 	return []string{"Open", "Close"}[fs]
 }
 
-func (*filePlugin) Init(ctx context.Context, e eng.Engine) bool {
+func (*FilePlugin) Init(ctx context.Context, e eng.Engine) bool {
```

Comment:
I made this change when I reorganized some files to allow the plugins to be "compiled into" the runner binary.  The implementation of the plugin, such as `filePlugin` here, have to be capitalized and exported, like `FilePlugin`.